src/CoreWCF.Kafka/src/CoreWCF/Channels/KafkaQueueMessageContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using CoreWCF.Queue.Common;
+
+namespace CoreWCF.Channels;
+
+public class KafkaQueueMessageContext : QueueMessageContext
+{
```

Comment:
Do you need this class when QueueMessageContext has a Properties property which you've fixed the lazy constructor for?

Reply:
I no longer need it indeed. good catch.